File: WindowsFramework/WindowsFramework.Entity/Action/User/LoginRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFramework.Entity.Action.User
{
    public class LoginRequest : Request, IRequest
    {
        public string LoginName { get; set; }
        public string LoginPasword { get; set; }
    }

    public class LoginResponse : Response, IResponse
    {
        public string LoginName { get; set; }
        public Guid UserId { get; set; }
        public string UserName { get; set; }
    }
}

File: WebFramework3/WebApi/src/Entity/Application/User/LoginRequest.java
package Entity.Application.User;

import Entity.Application.IRequest;
import Entity.Application.Request;

/**
 * Created by Bianshuai on 2017/1/8.
 */
public class LoginRequest extends Request implements IRequest {

    //登录名
    public String LoginName = null;

    //登录密码
    public String LoginPassword = null;
}
